### PR TITLE
Add CodeChecker to the command line tools

### PR DESCRIPTION
**Pull request by martong**

### CODE REVIEW

File: clang/www/analyzer/command-line.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<div id="page">
+<!--#include virtual="menu.html.incl"-->
+<div id="content">
+
+<h1>Running the analyzer from the command line</h1>
+
+<p>The following tools are supported to run the analyzer from the command line:
+<ul>
+  <li><a href="https://github.com/Ericsson/codechecker">CodeChecker</a> (preferred on Linux)</li>
```
Comment (gyorb) [inline clang/www/analyzer/command-line.html]: Maybe a separate minimal html page would be great for CodeChecker too. Just some basic steps with an example to get a feeling about the usage.
For a more detailed description the user can go to the github page.

Comment (gyorb) [inline clang/www/analyzer/command-line.html]: Maybe we should just list the supported platforms, I'm not sure the `preferred on Linux` will be accepted but we will see if we keep it.

File: clang/www/analyzer/command-line.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <li><a href="scan-build.html">scan-build</a> (preferred on macOS)</li>
+</ul>
+</p>
+
+<p>
+Both tools are wrapper scripts to drive the analysis and the underlying invocations of the Clang compiler.
```
Comment (gyorb) [inline clang/www/analyzer/command-line.html]: CodeChecker runs clang-tidy too

**Comment by martong** [comment]
Ping @dkrupp

**Comment by martong** [comment]
Ping @dkrupp @gyorb

**Comment by whisperity** [comment]
@martong Is this still relevant? Didn't we merge something similar to upstream?

**Comment by martong** [comment]
> @martong Is this still relevant? Didn't we merge something similar to upstream?

Yes we did, but we used this PR as a pre-review before going to Phab.

**Pull request closed without merging**